staging/src/k8s.io/client-go/tools/clientcmd/loader_test.go

```diff
@@ -201,6 +201,170 @@ func TestLoadingEmptyMaps(t *testing.T) {
 	}
 }
 
+func TestDuplicateClusterName(t *testing.T) {
+	configFile, _ := ioutil.TempFile("", "")
+	defer os.Remove(configFile.Name())
+
+	err := ioutil.WriteFile(configFile.Name(), []byte(`
+kind: Config
+apiVersion: v1
+clusters:
+- cluster:
+    api-version: v1
+    server: https://kubernetes.default.svc:443
+    certificate-authority: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+  name: kubeconfig-cluster
+- cluster:
+    api-version: v1
+    server: https://kubernetes.default.svc:443
+    certificate-authority: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
```

Comment:
Could you change the api-version, server, and certificate-authority? We want to make sure an error is returned even if `name` is the only duplicate bit.

Same for the other tests.